### PR TITLE
Reworded the Why conda-forge section

### PR DESCRIPTION
**Pull request by bashirk**
PR Checklist:

- [x] make all edits to the docs in the `src` directory, not in `docs`
- [x] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] put any other relevant information below

This PR fixes issue #1307

### CODE REVIEW

File: src/user/introduction.rst [hunk 1]
```diff
@@ -21,7 +21,7 @@ You can `search <https://anaconda.org/>`__ for packages online. Look out for pac
 Why conda-forge?
 ----------------
 
-The conda team from Anaconda, Inc packages a multitude of packages and provides them to all users free of charge in their ``default`` channel.
+The conda team, from `Anaconda Inc. <https://anaconda.org/>`__, packages a multitude of packages and provides them to all users free of charge in their ``default`` channel.
```
Comment (leofang) [inline src/user/introduction.rst:24]:
```suggestion
The conda team, from `Anaconda, Inc. <https://anaconda.org/>`__, packages a multitude of packages and provides them to all users free of charge in their ``default`` channel.
```

Comment (viniciusdc) [reply src/user/introduction.rst:24]: Great, thanks @leofang!

**Comment by viniciusdc** [comment]
LGTM @bashirk , once you commit that suggested changes we are ready :smile:

**Review by viniciusdc — COMMENTED** [review]
LGTM

**Pull request merged**